pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -127,7 +127,7 @@ assert withCryptsetup -> (cryptsetup != null);
 let
   wantCurl = withRemote || withImportd;
   wantGcrypt = withResolved || withImportd;
-  version = "250.4";
+  version = "251.2";
 
   # Bump this variable on every (major) version change. See below (in the meson options list) for why.
   # command:
```

Comment:
The releaseTimestamp below should be `1653139508`

Reply:
updated.